.eslintignore

```diff
@@ -2,3 +2,5 @@
 static/legacy/
 external/
 build/
+# Top level await isn't supported till ESLint 8
+locale/en/blog/release/v17.0.0.md
```

Comment:
Are we blocked because standard is still using eslint 7.x?

I think I've expressed my opinion in the past, standard does not catch many of real issues and personally I never use it. prettier doesn't catch real issues either.

Just thinking out loud for the future :)

Reply:
Yeah, I believe the ESLint upgrade is blocked on prettier, but eslint-plugin-promise just released a compatible version, so that might have been the last one remaining

Reply:
All good, as long as someone remembers to revert this later :)